src/flavors/raingardens/config.yml

```diff
@@ -130,7 +131,7 @@ map:
 # The values map the place type to map icons (defined below).
 place_types:
   raingarden:
-    label: _(Raingarden)
+    label: _(raingarden)
```

Comment:
Any reason why this is was lower cased? Our other place types, like `Observation`, `Idea` etc are capitalized. Whatever we decide, I think we should maintain consistency across all of our place types.